### PR TITLE
Feat/Intro to staking page

### LINKED ISSUE #646: New intro to staking page

### PR DESCRIPTION
**Pull request by sam-keen**
# Related issues 🔗

Closes #646

# Description ℹ️

Provides new staking intro as per 646. Does not include:

- New validators table design (separate ticket)
- Dismissible intro section 

# Demo 📺

Screenshot 2022-07-11 at 11 25 34

### CODE REVIEW

**Review by dexturr — APPROVED** [review]

**Review by mattrussell36 — APPROVED** [review]

File: apps/token/src/routes/staking/staking.tsx [hunk 1]
```diff
@@ -1,224 +1,50 @@
...
 };
 
-export const StakingStepConnectWallets = () => {
-  const { ETHERSCAN_URL } = useEnvironment();
-  const { t } = useTranslation();
-  const { account } = useWeb3React();
-  const { keypair } = useVegaWallet();
-  const { appDispatch } = useAppState();
-
-  if (keypair && account) {
```
Comment (dexturr) [inline apps/token/src/routes/staking/staking.tsx:81]: question (non-blocking): just validating we are getting rid of the connect states on this page? I guess it makes sense as they need to connect on the page they are using, but just wanted to confirm

File: apps/token/src/i18n/translations/dev.json [hunk 2]
```diff
@@ -512,10 +512,11 @@
...
   "unknownEthereumConnectionError": "An unknown error occurred. Check the console in your browser's web developer tools for more details",
-  "stakingDescription1": "Associate VEGA tokens from your Ethereum wallet to a Vega wallet/key to nominate a validator",
-  "stakingDescription2": "Validators and their nominators are rewared every 24 hours for securing the network with a cut of trading fees (and rewards from the Vega treasury). Staking rewards are paid into your Vega wallet after an epoch ends.",
-  "stakingDescription3": "The amount that a validator, and its nominators receives is proportionate to the amount staked on it, however a validator can be penalised during an epoch for a number of reasons. In this case the validator and its nominator will forfeit some or all of its reward from that epoch",
-  "stakingDescription4": "A validator will receive a higher share of the pot than its nominators. This share is the same for all validators and is set by a network parameter.",
+  "stakingDescriptionTitle": "How does staking on Vega work?",
+  "stakingDescription1": "1. VEGA is an ERC20 token. Associate it with a Vega wallet using a staking bridge",
```
Comment (candida-d) [inline apps/token/src/i18n/translations/dev.json:516]:
```suggestion
  "stakingDescription1": "1. VEGA is an ERC20 token. Associate it with a Vega wallet using the staking bridge",
```

File: apps/token/src/i18n/translations/dev.json [hunk 3]
```diff
@@ -512,10 +512,11 @@
...
+  "stakingDescription2": "2. Use your Vega wallet to nominate a validator",
```
Comment (candida-d) [inline apps/token/src/i18n/translations/dev.json:517]: Comment (semi-non-blocking?): Really you're using the token dApp to nominate a validator as you can't do it from within the wallet. But am I missing the gist of what this is trying to convey?

File: apps/token/src/routes/staking/staking.tsx (hunk 1 above)
Comment (sam-keen) [reply apps/token/src/routes/staking/staking.tsx:81]: That's right, it's a much simpler page now.

File: apps/token/src/i18n/translations/dev.json (hunk 3 above)
Comment (candida-d) [reply apps/token/src/i18n/translations/dev.json:517]: That’s clearer! @campbellssource @sam-keen

**Pull request merged**